src/components/PhotoSwipe/PhotoSwipe.tsx

```diff
@@ -528,6 +553,91 @@ function PhotoSwipe(props: Iprops) {
         }
     }, [showInfo]);
 
+    useEffect(() => {
+        if (!isOpen) return;
+        const item = items[photoSwipe?.getCurrentIndex()];
+        if (item && item.metadata.fileType === FILE_TYPE.LIVE_PHOTO) {
+            let videoPlaying = false;
+
+            const playVideo = async (livePhotoVideo, livePhotoImage) => {
+                if (videoPlaying) return;
+                videoPlaying = true;
+                livePhotoVideo.style.opacity = 1;
+                livePhotoImage.style.opacity = 0;
+                livePhotoVideo.load();
+                livePhotoVideo.play().catch(() => {
+                    pauseVideo(livePhotoVideo, livePhotoImage);
+                });
+            };
+
+            const pauseVideo = async (livePhotoVideo, livePhotoImage) => {
```

Comment:
we can move pauseVideo and playVideo function outside the `useEffect` i think to util i think